app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,8 @@
         android:label="@string/app_name"
         android:theme="@style/LightThemeSelector"
         android:allowBackup="true"
-        android:networkSecurityConfig="@xml/network_security_config">
+        android:networkSecurityConfig="@xml/network_security_config"
+        android:localeConfig="@xml/locales_config">
```

Comment:
What does this change do? Probably not related to v15? A separate commit would be appreciated 🙂

Reply:
It signals to the OS which languages the app is localized in. This allows you to set a language on a per-app basis in Android >= 13.

I added this mainly so that I could test the German localization without changing my system's overall language. No, it's not related to v15.

Reply:
Reverted.

Reply:
Ah, reverting wouldn't have been necessary. The change is nice, do you want to open a separate PR for it?

Reply:
Yes: #55